packages/ketcher-react/src/script/editor/shared/closest.ts

```diff
@@ -285,9 +310,9 @@ function findClosestEnhancedFlag(restruct, pos, skip, _minDist, options) {
   return ret;
 }
 
-function findClosestDataSGroupData(restruct, pos) {
-  let minDist = null;
-  let ret = null;
+function findClosestDataSGroupData(restruct: ReStruct, pos: Vec2) {
+  let minDist = Number.POSITIVE_INFINITY;
```

Comment:
Why not to keep null here? I'm not sure that it will not affect some functionality

Reply:
Because with types we need to additionally check whether this field is null. We do not return minDist across this file so this is safe to update